src/main/java/com/mariamkatamashvlii/gym/service/implementation/TraineeServiceImpl.java

```diff
@@ -284,21 +273,40 @@ public List<TrainerDTO> updateTrainers(UpdateTrainersRequestDTO updateTrainersRe
             newTrainers.add(dto);
         });
 
-        log.info("[{}] Successfully updated trainer list for trainee: {}", transactionId, username);
         return newTrainers;
     }
 
     @Override
     @Transactional
     public void toggleActivation(ToggleActivationDTO toggleActivationDTO) {
-        String transactionId = UUID.randomUUID().toString();
         String username = toggleActivationDTO.getUsername();
         validator.validateUserExists(username);
-        log.info("[{}] Attempting to toggle activation for user: {}", transactionId, username);
 
         User user = userRepo.findByUsername(username);
         user.setIsActive(toggleActivationDTO.getIsActive());
         userRepo.save(user);
-        log.info("[{}] Successfully toggled activation for user: {}. Now active: {}", transactionId, username, user.getIsActive());
+    }
+
+    private void removeTrainings(String username) {
```

Comment:
This method belongs more to another service like TrainingService. So, then it should be subsequently called from API layer adter trainee has been deleted